source/_integrations/device_tracker.markdown

```diff
@@ -99,20 +99,28 @@ In the example above, `devicename` refers to the detected name of the device.  F
 | `track`         | [uses platform setting]       | If  `yes`/`on`/`true` then the device will be tracked. Otherwise its location and state will not update.                                                                                                                                                                                                                                        |
 | `consider_home` | [uses platform setting]       | Seconds to wait till marking someone as not home after not being seen. Allows you to override the global `consider_home` setting from the platform configuration on a per device level.                                                                                                                                                         |
 
-## Device states
+## The state of a tracked device
 
-The state of your tracked device will be `'home'` if it is in the [home zone](/integrations/zone#home-zone), detected by your network or Bluetooth based presence detection. If you're using a presence detection method that includes coordinates then when it's in a zone the state will be the name of the zone (case sensitive). When a device isn't at home and isn't in any zone, the state will be `'not_home'`.
+A device tracker entity can have one of two states: **Home**, or **Not home**.
```

Comment:
A device tracker, with GPS as source, can have any number of string states, since the integration can either report GPS coordinates which will be matched to a zone which is set as state or report a location which could be any string (which could match a zone) which is set as state.

A device tracker with Router as source will only have home/not_home states.

https://developers.home-assistant.io/docs/core/entity/device-tracker